easybuild/tools/github.py

```diff
@@ -351,6 +371,9 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
     patched_files = det_patched_files(txt=diff_txt, omit_ab_prefix=True, github=True, filter_deleted=True)
     _log.debug("List of patched files: %s" % patched_files)
 
+    for key, val in sorted(pr_data.items()):
+        _log.debug("\n%s:\n\n%s\n" % (key, val))
```

Comment:
if this will be retained, make the log messages a bit more descriptive (mention what is being printed here)